### ISSUE TITLE
Expand on readme details

### ISSUE THREAD

**Issue by pieroway** [marcules/karma-remap-istanbul#28]
"You will need to either install karma-coverage and configure it as a preprocessor for your transpiled modules under test or instrument the modules under test as part of your build process. If the latter option is chosen, the coverage statistics will need to be stored at the __coverage__ global variable (istanbul's default) or karma will not transmit them back to the runner."

Can you find the time to better describe what this means in plain English? :)
As written, someone like me who is relatively new to Angular2 and Typescript, will get lost with that statement.

**Comment by mattlewis92** [comment]
I've tried to explain the paragraph a bit more, if this is still hard to follow, please let me know! 😃
